Sources/PackageDescription/PackageDescription.docc/Curation/Extensions/LanguageTag-notEqual.md

```diff
@@ -0,0 +1,15 @@
+# ``PackageDescription/LanguageTag/!=(_:_:)``
+
+@Metadata {
+   @DocumentationExtension(mergeBehavior: override)
+}
+
+Returns a Boolean value indicating whether two values are not equal.
+
+- Parameters:
+  - lhs: A value to compare.
+  - rhs: Another value to compare.
+
+Inequality is the inverse of equality. For any values a and b, a != b implies that a == b is false.
+
+This is the default implementation of the not-equal-to operator (!=) for any type that conforms to Equatable.
```

Comment:
Related to a comment i left before: I don't think it adds value to copy-past the default implementation's documentation. If there is anything unique, then we should _customize_ the documentation. If there is nothing to customize, then we should  just let the documentation automatically be inherited from the default implementations; otherwise, the copy-pasted documentation can become silently outdated when the default  documentation's changes.

Reply:
The same opinion applies to the other PR too.